PointOfDispensingApp/src/components/VaccinationEventPage.vue

```diff
@@ -26,21 +26,42 @@ import PatientInfoComponent from './PatientInfoComponent';
 import VaccinationScreeningComponent from './VaccinationScreeningComponent';
 import VaccinationProceedComponent from './VaccinationProceedComponent';
 import VaccinationCanceledComponent from './VaccinationCanceledComponent';
+import config from '../../config.js';
 
   export default {
     name: 'VaccinationEventPage',
     methods: 
     {
+      setVaccinationProceedDecision(decision)
+      {
+        if(decision == true)
+        {
+          this.vaccinationProceedDecision = config.vaccinationDecisionState.VACCINATION_PROCEED_YES
+        }
+        else
```

Comment:
Code wise, you can reduce the number of lines with a ternary here if you'd like 
```
this.vaccinationProceedDecision = Decision 
    ? config.vaccinationDecisionState.VACCINATION_PROCEED_YES 
    : config.vaccinationDecisionState.VACCINATION_PROCEED_NO
```

This assumes `decision` is a bool. Even if it isn't, you could still use the ternary by saying `decision == true` in the operator. 

If you'd rather not use a ternary, `if(decision)` should have the same effect.

Reply:
Ok!  updated to use ternary